Oglle/dist/presence.js

```diff
@@ -1,50 +1,60 @@
 var presence = new Presence({
-  clientId: "641614088215986186"
+    clientId: "641614088215986186",
+    mediaKeys: false
```

Comment:
it was good like it was before, mediakeys are removed, so should no longer be added.